docs/howto/backup-and-restore.md

```diff
@@ -0,0 +1,146 @@
+
+* Python example to create a bare lakeFS repository using S3 storage:
+
+```python
+lakefs.Repository("target-example-repo").create(storage_namespace="s3://target-bucket-name/example-repo", default_branch="same-default-branch-as-in-source-repo", bare=True)
+```
+
+* Python example to create a bare lakeFS repository using Azure storage:
+
+```python
+lakefs.Repository("target-example-repo").create(storage_namespace="https://target-storage-account-name.blob.core.windows.net/targetContainer/example-repo", default_branch="same-default-branch-as-in-source-repo", bare=True)
+```
+
+* [lakeFS CLI](https://docs.lakefs.io/reference/cli.html#lakectl-repo-create-bare) command to create a bare lakeFS repository using S3 storage:
+
+```
+lakectl repo create-bare lakefs://target-example-repo s3://target-bucket-name/example-repo --default-branch "same-default-branch-as-in-source-repo"
+```
+
+* [lakeFS CLI](https://docs.lakefs.io/reference/cli.html#lakectl-repo-create-bare) command to create a bare lakeFS repository using Azure storage:
+
+```
+lakectl repo create-bare lakefs://target-example-repo https://target-storage-account-name.blob.core.windows.net/targetContainer/example-repo --default-branch "same-default-branch-as-in-source-repo"
+```
+
+### Restore Metadata to new Repository
+Run restore_refs to load back all commits, tags and branches.
+
+* Python example to restore metadata to new repository. First download metadata(refs_manifest.json) file created by metadata dump process:
+
+```
+aws s3 cp s3://target-bucket-name/example-repo/_lakefs/refs_manifest.json .
+```
+
+```
+azcopy copy 'https://target-storage-account-name.blob.core.windows.net/targetContainer/example-repo/_lakefs/refs_manifest.json?<target_container_SAS_token>' .
+```
+
+Then read refs_manifest.json file and restore metadata to new repository:
+```python
+with open('./refs_manifest.json') as file:
+    refs_manifest_json = json.load(file)
+    print(refs_manifest_json)
+    
+target_lakefs_sdk_client.internal_api.restore_refs(target_repo_name, refs_manifest_json)
+```
+
+* [lakeFS CLI](https://docs.lakefs.io/reference/cli.html#lakectl-refs-restore) command to restore metadata to new repository using S3 storage:
+
+```
+aws s3 cp s3://target-bucket-name/example-repo/_lakefs/refs_manifest.json - | lakectl refs-restore lakefs://target-example-repo --manifest -
+```
+
+* [lakeFS CLI](https://docs.lakefs.io/reference/cli.html#lakectl-refs-restore) command to restore metadata to new repository using Azure storage:
+
+```
+az storage blob download --container-name targetContainer --name example-repo/_lakefs/refs_manifest.json --account-name target-storage-account-name --account-key <target-storage-account-key> | lakectl refs-restore lakefs://target-example-repo --manifest -
+```
+
+**Note:** If you are running backups regularly then test the restore process periodically to make sure that you can restore the repository in case of any disaster.
```

Comment:
```suggestion
**Note:** If you are running backups regularly it is highly advised to test the restore process periodically to make sure that you are able to restore the repository in case of disaster.
```